project/database/connection.py

```
@@ -0,0 +1,44 @@
+from sqlalchemy.orm import Session, sessionmaker
+
+# Database URL from environment
+DATABASE_URL = os.getenv("DATABASE_URL", "sqlite:///warnings.db")
+
+# SQLAlchemy setup
+engine = create_engine(
+    DATABASE_URL,
+    # SQLite specific settings
+    connect_args={"check_same_thread": False} if "sqlite" in DATABASE_URL else {},
+    # PostgreSQL specific settings
+    pool_pre_ping=True if "postgresql" in DATABASE_URL else False,
```

Ruff (0.12.2)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)